addons/sale/models/sale_order_line.py

```diff
@@ -1173,10 +1173,14 @@ def _validate_analytic_distribution(self):
 
     #=== CORE METHODS OVERRIDES ===#
 
+    def _get_partner_display(self):
+        self.ensure_one()
+        commercial_partner = self.order_partner_id.commercial_partner_id
+        return f'({commercial_partner.ref or commercial_partner.name})'
+
     def _additional_name_per_id(self):
         return {
-            so_line.id:
-            '(%s)' % (so_line.order_partner_id.ref or so_line.order_partner_id.name)
+            so_line.id: so_line._get_partner_display()
             for so_line in self
             if so_line.order_partner_id.ref or so_line.order_partner_id.name
```

Comment:
Shouldn't this conditional line be changed as well? It made sense before, using the `order_partner_id`, but now, it looks strange to me.